src/worker/abstract-worker.ts

```diff
@@ -95,7 +90,6 @@ export abstract class AbstractWorker<
         this.mainWorker = value.parent
       } else if (value.kill) {
         // Here is time to kill this worker, just clearing the interval
```

Comment:
Do we need this comment? If yes, should it be another sentence or is it still correct?

Reply:
It is a correct statement